src/components/mdIcon/mdIcon.vue

```diff
@@ -17,7 +19,8 @@
 
   export default {
     props: {
-      mdSrc: String
+      mdSrc: String,
+      mdIconset: Boolean
```

Comment:
seems to work very very well:
![image](https://cloud.githubusercontent.com/assets/556695/21876803/e7e2467e-d864-11e6-810b-fa5705225d2a.png)

```
<md-icon md-iconset="fa fa-home"></md-icon>
<md-icon md-iconset="fa fa-bars" class="md-size-2x md-primary"></md-icon>
<md-icon md-iconset="fa fa-users" class="md-size-3x md-accent"></md-icon>
<md-icon md-iconset="fa fa-clock-o" class="md-size-4x md-warn"></md-icon>
```